server.js

```diff
@@ -15,8 +15,16 @@ const formatError = require('./utils/format-error');
 const { currencyWorker } = require('./currency.worker');
 const formatErrorForLogger = require('./utils/format-error-for-logger');
 const { dotenvVariables } = require('./dotenvValidator');
+const {
+  SUPER_ADMIN_EMAIL,
+  SUPER_ADMIN_PASSWORD,
+  NODE_ENV,
+} = require('./dotenvValidator');
+const { registerAdmin } = require('./__tests__/helper-functions');
 
-connectDB();
+if (NODE_ENV === 'test') {
+  registerAdmin(SUPER_ADMIN_EMAIL, SUPER_ADMIN_PASSWORD);
```

Comment:
why you remove this?

Reply:
We have 2 connections to database: in `app.js` and `server.js`, so I remove one of them.

Reply:
ok